pkg/hook/hook_manager.go

```diff
@@ -201,7 +201,7 @@ func (hm *Manager) loadHook(hookPath string) (hook *Hook, err error) {
 	hook.WithTmpDir(hm.TempDir())
 
 	if hook.Config == nil {
-		return nil, fmt.Errorf("hook %q is marked as executable but doesn't contain config section", hook.Path)
+		return nil, fmt.Errorf("hook %s is marked as executable but doesn't contain config section", hook.Path)
```

Comment:
Why are we changing %q to %s here? Thus, the hook name will now be without quotes.

Reply:
you're right